File: cucumber-deltaspike/src/test/java/io/cucumber/deltaspike/RunCucumberTest.java
package io.cucumber.deltaspike;

import org.junit.platform.suite.api.ConfigurationParameter;
import org.junit.platform.suite.api.IncludeEngines;
import org.junit.platform.suite.api.SelectClasspathResource;
import org.junit.platform.suite.api.Suite;

import static io.cucumber.junit.platform.engine.Constants.GLUE_PROPERTY_NAME;

@Suite
@IncludeEngines("cucumber")
@SelectClasspathResource("io/cucumber/deltaspike")
@ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "io.cucumber.deltaspike")
public class RunCucumberTest {

}

File: kotlin-java8/src/test/kotlin/io/cucumber/kotlin/RunCucumberTest.kt
package io.cucumber.kotlin

import io.cucumber.junit.platform.engine.Constants
import org.junit.platform.suite.api.ConfigurationParameter
import org.junit.platform.suite.api.IncludeEngines
import org.junit.platform.suite.api.SelectClasspathResource
import org.junit.platform.suite.api.Suite

@Suite
@IncludeEngines("cucumber")
@SelectClasspathResource("io/cucumber/kotlin")
@ConfigurationParameter(key = Constants.GLUE_PROPERTY_NAME, value = "io.cucumber.kotlin")
class RunCucumberTest
